addons/website_event_track/controllers/event_track.py

```diff
@@ -153,7 +153,7 @@ def _event_tracks_get_values(self, event, tag=None, **searches):
         for tracks_group in tracks_by_day:
             # the tracks group is folded if all tracks are done (and if it's not "today")
             tracks_group['default_collapsed'] = (today_tz != tracks_group['date']) and all(
-                track.is_track_done and not track.is_track_live
+                not track.is_track_done if search_tags else track.is_track_done and not track.is_track_live
```

Comment:
If I understand the logic: if there are search tags, we collapse the group if all tracks occur another day than today and if all tracks are not marked as done. **Is it what Pierre wants?**

If there is search tag, I think we should rather collapse the group if all tracks occur in the future (`(today_tz < tracks_group['date'])`) check or if there is no track matching with the given search tags or search term.

Reply:
Correct me if I'm wrong, but based on the initial specs, the intention was to expand the groups that are already done when there are search tags (filtering) and collapse them when there's no filtering. It seems like Pierre wants to expand the groups that are already done during filtering while keeping future groups collapsed. Could you please clarify if you'd like to collapse groups where all tracks occur in the future and also collapse groups that are already done?

Reply:
Rethinking of it: It's not really clear to me. I think we'd better ask Pierre.

I suggested the other folding strategy because it allows people to easily find information about the speaker who hosted the talk they attended (see: spec).

But, we may unfold too much talks and add noise around the today's talk. Maybe, we should rather unfold all sections if (1) all sections are folded and if (2) the user clicked on a search tag.

Reply:
It seems that the SAVP is on leave until the 13th of September, so perhaps we could consult LNA on the matter given the task's priority. However, expanding the groups while applying the filter (option 2) seems to make more sense. Or we can keep it as is following the current specs? What do you think? :thinking:

Reply:
We can leave it like that and see if it's OK during testing :smiley: